devfiles/isce3/devfile/devfile.yaml

```diff
@@ -55,8 +55,12 @@ components:
                     mountPath: /projects/shared-buckets
                     subPath: shared-buckets
                     mountPropagation: HostToContainer
+                  - name: s3fs-volume
+                    mountPath: /projects/triaged-jobs
+                    subPath: triaged-jobs
+                    mountPropagation: HostToContainer
               - name: s3fs
-                image: 'mas.ops.maap-project.org/root/che-sidecar-s3fs:master'
+                image: 'mas.dit.maap-project.org/root/che-sidecar-s3fs:master'
```

Comment:
Should this be dit? @bsatoriu

Reply:
Yes

Reply:
How does ops get updated to the .ops. sidecar image?